tap_redshift/__init__.py

```diff
@@ -474,6 +479,32 @@ def build_state(raw_state, catalog):
     return state
 
 
+def get_column_orders():
+    parser = argparse.ArgumentParser()
+
+    # Only added arguments needed for running extraction command.
+    parser.add_argument('-c', '--config')
+    parser.add_argument('-p', '--properties')
+    parser.add_argument('--catalog')
+
+    args = parser.parse_args()
+
```

Comment:
utils.parse_args casts catalog to Catalog class, dropping column_order_map (our custom property).